advanced/Scripts/piholeDebug.sh

```diff
@@ -779,6 +779,26 @@ check_required_ports() {
     done
 }
 
+ip_command() {
+    # Obtain and log information from "ip XYZ show" commands
+    echo_current_diagnostic "${2}"
+    OLD_IFS="$IFS"
+    IFS=$'\n'
```

Comment:
You don't really need to save IFS and restore it. Changing it inside a function just affects the time inside the function.

Reply:
Thanks, I just copy-pasted from another function (DHCP discovery), I will just make `IFS` local :+1:

Reply:
Actually, `IFS` is already `\n` by default. I don't think we really have to set it explicitly.